infra/src/main/java/com/redhat/service/smartevents/infra/api/models/responses/ErrorResponse.java

```diff
@@ -6,8 +6,11 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.redhat.service.smartevents.infra.api.APIConstants;
 import com.redhat.service.smartevents.infra.exceptions.BridgeError;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(name = "Error",
```

Comment:
And here

Reply:
@wtrocki am I correct here, for the `Error`, is the answer same as for `ObjectReference`?

Reply:
yes